example/lib/main.dart

```diff
@@ -66,6 +69,19 @@ class Storyboard extends StatelessWidget {
                     );
                   },
                 ),
+                ComponentState(
+                  docs: '''
+                    IconButton(onPressed: ())
```

Comment:
Should this not include the param `icon`?

Reply:
correct